forge-gui/res/cardsfolder/upcoming/vesuvan_duplimancy.txt

```diff
@@ -0,0 +1,9 @@
+Name:Vesuvan Duplimancy
+ManaCost:3 U
+Types:Enchantment
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl  | Execute$ TrigCopyTarget | ValidActivatingPlayer$ You | IsSingleTarget$ True | TriggerZones$ Battlefield | TargetsValid$ Creature.YouCtrl,Artifact.YouCtrl | RememberValidCards$ True  |  TriggerDescription$ Whenever you cast a spell that targets only a single artifact or creature you control, create a token that's a copy of that artifact or creature, except it's not legendary 
```

Comment:
Did you test this with stuff like _Unearth_?

Because it doesn't say card it shouldn't trigger, so you might need `inZoneBattlefield`

Reply:
There's also extra spaces where there shouldn't be.